config/jobs/kubernetes/cloud-provider-azure/cloud-provider-azure-config.yaml

```diff
@@ -164,7 +164,6 @@ periodics:
     preset-service-account: "true"
     preset-cloudprovider-azure-cred: "true"
     preset-dind-enabled: "true"
-  decorate: true
```

Comment:
what does `decorate` mean originally?

Reply:
it means setup the command instead of using the image's one, we don't need this (it actually make our tests failing).